Makefile

```diff
@@ -189,6 +189,7 @@ delete-tag:
 
 .PHONY: docker-otelcol
 docker-otelcol:
+	cp examples/otel-local-config.yaml cmd/otelcol
```

Comment:
This is not nice but seems better than setting the docker context to the root of the project.